### PR TITLE
New template for webfarm and NSG

### PR DESCRIPTION
**Pull request by s-malone**
Includes VNET, LB, PublicIP, NICs, VMs, Inbound NAT rules, Network
Security Groups and DSC to enable a website

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@s-malone__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (smalone). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: 101-networkroutes/azuredeploy-parameters.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "namePrefix": {
+      "value": ""
```
Comment (singhkay) [inline 101-networkroutes/azuredeploy-parameters.json]: Fill in a dummy value for the validation bot

File: 101-networkroutes/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Secure public load-balanced website on Windows IIS
```
Comment (singhkay) [inline 101-networkroutes/README.md]: @s-malone Can you rename this folder to 101-network-routes so it reads/looks better?

File: 101-networkroutes/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        }
+    },
+    "variables": {
+        "routeTable1Name": "[concat(parameters('namePrefix'), 'RouteTable1')]",
+        "routeTable2Name": "[concat(parameters('namePrefix'), 'RouteTable2')]",
+        "virtualNetworkName": "[concat(parameters('namePrefix'), 'VNet')]"
+    },
+    "resources": [
+		{
+			"apiVersion": "2015-06-15",
```
Comment (singhkay) [inline 101-networkroutes/azuredeploy.json]: Would probably be a good idea to start making apiVersion a variable to make it easy to move to a newer api version

**Comment by singhkay** [comment]
@s-malone Any update on this?

**Comment by singhkay** [comment]
@s-malone Closing since I haven't heard in a while on the state of this template

**Pull request closed without merging**